frontend/lib/js/form-components/filterTypes.js

```diff
@@ -14,6 +15,7 @@ export const SUPPORTED_FILTERS = {
   REAL_RANGE: REAL_RANGE,
   STRING: STRING,
   BIG_MULTI_SELECT: BIG_MULTI_SELECT,
+  MONEY_RANGE: MONEY_RANGE,
   // Maybe later:
   // 'MONEY_RANGE'
```

Comment:
Nitpick: Could you remove this comment line, now that we have a `MONEY_RANGE`? Could you also add `MONEY_RANGE` below `REAL_RANGE` above, so that all range filter types are grouped together?